### PR TITLE
Fix typo, add Calibre and Tailscale.

### PR DESCRIPTION
**Pull request by enhancedrock**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -309,6 +311,7 @@
 [Suwayomi](https://github.com/Suwayomi/Suwayomi-Server) :c-selfhost: - Manga reader server that runs extensions built for Tachiyomi. Clients are available on their readme  
 [Kotatsu](https://github.com/KotatsuApp/Kotatsu) :c-android: | [TachiyomiSY](https://github.com/jobobby04/TachiyomiSY) :c-android: | [Neko](https://github.com/nekomangaorg/Neko) :c-android: | [Mihon](https://mihon.app/) :c-android: - Android manga readers. TachiyomiSY, Neko and Mihon are forks of [Tachiyomi](https://tachiyomi.org/)  
 [Paperback](https://paperback.moe/) :c-ios: - Ad-free Manga reader for iOS
+[Calibre](https://calibre-ebook.com/) :c-selfhost: :c-pc: - E-Book collection manager, metadata editor, e-book editor, e-book viewer (with integrated web server for viewer), news/magazine downloader with built-in backups. Some e-reading apps also include the ability to connect to the web server for viewing content.
```
Comment (resucutie) [inline index.md]: [I saw something about a weird integration with mobile clients](https://calibre-ebook.com/download_android). Could you explain, if you have any knowledge?

File: index.md [hunk 2]
```diff
@@ -674,4 +677,4 @@ Merged entries:
 Thanks to:
 - [FontAwesome](https://fontawesome.com/) and [Material Icons](https://fonts.google.com/icons) for providing the icons
 - [Volpeon](https://volpeon.ink/) for the original [Blobfox](https://volpeon.ink/emojis/blobfox/) in a box icon (we modified it a lil bit to change the icon inside da box :3)
-- [Ripped](https://ripped.guide/) as the inspiration of this project
\ No newline at end of file
+- [Ripped](https://ripped.guide/) as the inspiration of this project
```
Comment (resucutie) [inline index.md:678]: Why was this moved?

**Review by resucutie — COMMENTED** [review]
Some things to explain

**Comment by resucutie** [comment]
frick. apparently idk how to use github

File: index.md [hunk 3]
```diff
@@ -269,6 +268,8 @@
 
 [Terms of Service; Didn't read](https://tosdr.org/) :c-extension: :c-resource: - Resource to get a quick summary on terms of service, as well as an overall privacy and user freedom score
 
+[Tailscale](https://tailscale.com/) :c-pc: :c-mobile: :c-freemium: - Zero-config WireGuard VPN to remotely access a device's services anywhere, with a dedicated web SSH client for devices advertising as Tailscale SSH available, can also advertise as an 'Exit Node' on some devices to route internet traffic through said Exit Node.
```
Comment (resucutie) [inline index.md]: 1.  Could we remove the part "advertising as Tailscale SSH available, can also advertise as an 'Exit Node' on some devices to route internet traffic through said Exit Node."?

2. Tailscale can be selfhostable? If yes, then add the proper icon

**Review by resucutie — CHANGES_REQUESTED** [review]

File: index.md (hunk 1 above)
Comment (enhancedrock) [reply index.md]: these are just ways calibre can interface with mobile devices, but some dedicated e-reader apps (not calibre) can load your content from the calibre content server, will clarify that

File: index.md (hunk 2 above)
Comment (enhancedrock) [reply index.md:678]: didn't mean to

File: index.md (hunk 3 above)
Comment (enhancedrock) [reply index.md]: 1. why?
2. nope, at least if it can only for enterprises

**Review by resucutie — APPROVED** [review]

File: index.md (hunk 3 above)
Comment (resucutie) [reply index.md]: 1. too technical. simplicity is good, after all the one that will read doesn't know anything about tailscale, hell what is a "exit node"? (don't respond, it is to illustrate)
2. so its a wireguard provider basically?

Comment (enhancedrock) [reply index.md]: yeah it is just a wireguard provider basically, you can just change it as you see fit but its basically just a way to remote access your devices over an advanced wireguard vpn called a 'tailnet', highly customisable

Comment (resucutie) [reply index.md]: well, i'll do changes over it. seems to go under the providers section

Comment (resucutie) [reply index.md]: @enhancedrock check the changes. anything that i missed?

Comment (enhancedrock) [reply index.md]: might be worth adding "designed with the purpose of easily remotely accessing (devices/servers/infrastructure - up to you)"?

Comment (resucutie) [reply index.md]: is it hard to access with wireguard? i thought tailscale was easy to configure, not easy to "access the stuff"

Comment (enhancedrock) [reply index.md]: it is easy to access stuff remotely, thats the main purpose of tailscale, it just does it through wiregurd

for example every tailscale device gets a:
tailscale ip like a LAN ip
a magic dns domain (for example surfboard-[magicdns-name].tailscale.com i cant remember the format exactly)
and devices keep their hostname so if you have `surfboard` bookmarked, going to `surfboard` on a diff network but connected to tailscale with that device also on the tailnet will still let you use that device through its hostname like normal

Comment (resucutie) [reply index.md]: i decided to remove tailscale. it is just a wireguard provider with features to make it easier due to its centralization, which for me the centralization alone makes it ring the red alarm on my head. i open an exception for proton vpn and mail because their business model relies on the consumer's trust, and their features shows that

also zerotier is selfhostable mostly (you cant host their UI from what i heard, but everything else you can), so we may look towards this instead
